test/integration-test/CodeGen/boundary_condition_2_benchmark.cpp

```diff
@@ -15,10 +15,10 @@
 //===------------------------------------------------------------------------------------------===//
 #define GRIDTOOLS_CLANG_GENERATED 1
 #define GRIDTOOLS_CLANG_HALO_EXTEND 3
-#include "gridtools/clang/verify.hpp"
 #include "test/integration-test/CodeGen/Options.hpp"
 #include "test/integration-test/CodeGen/generated/boundary_condition_2_c++-naive.cpp"
 #include "test/integration-test/CodeGen/generated/boundary_condition_2_gridtools.cpp"
+#include "gridtools/clang/verify.hpp"
```

Comment:
Hi Stefan, 

the GRIDTOOLS_CLANG_HALO_EXTEND is set above in line 17, so I dont see how reordering here changes the behaviour

Reply:
in that test case it is set. in other cases it is not there and we either have to set it or move the include. here i just did it to keep the include structure homogeneous among the integration tests.

Reply:
Ok, I see. 

I would opt for setting the includes everywhere in all the benchmarks as a solution. 
we can always forget it, and the bug is tricky to catch but with your PR 96, now it should be safe, isnt? 

Reason being is that relying on the ordering of the includes is not a very clean C++ practice, plus the clang-format will always mess around the order (as @twicki  pointed out)